spec/05-classes-and-objects.md

```diff
@@ -435,19 +439,20 @@ An early definition is type-checked and evaluated in the scope which
 is in effect just before the template being defined, augmented by any
 type parameters of the enclosing class and by any early definitions
 preceding the one being defined. In particular, any reference to
-`this` in the right-hand side of an early definition refers
+`this` in an early definition refers
 to the identity of `this` just outside the template. Consequently, it
-is impossible that an early definition refers to the object being
-constructed by the template, or refers to one of its fields and
+is impossible for an early definition to refer to the object being
+constructed by the template, or to refer to one of its fields and
 methods, except for any other preceding early definition in the same
```

Comment:
`fields or methods`?

Reply:
of the template.

Reply:
I meant s/and/or/ but now I'm not so sure.

Reply:
Ah, `s/and/and\/or/`.

Reply:
I would expand it to: "or to refer to one of its fields, or to one of its methods," but that begins to sound a bit High Church, albeit I like that.